corehq/apps/sms/views.py

```diff
@@ -1117,9 +1121,13 @@ def hq_api_id(self):
     @property
     @memoized
     def backend_class(self):
-        # Superusers can create/edit any backend
+        # Superusers and domain admins can create/edit any backend
         # Regular users can only create/edit Telerivet backends for now
-        if not self.is_system_admin and self.hq_api_id != SQLTelerivetBackend.get_api_id():
+        if (
+            not self.is_system_admin
+            and not self.is_domain_admin
+            and self.hq_api_id != SQLTelerivetBackend.get_api_id()
+        ):
```

Comment:
is this same as?
```
if not (
  self.is_system_admin
  or self.is_domain_admin
  or self.hq_api_id != SQLTelerivetBackend.get_api_id()
)
```
If yes,
that is easier to relate to with the comment.

Is there a need of a sign off on this change from product? I am not sure. Just thinking if it was kept that way for a reason.

Reply:
Think it should be
```py
if not (
    self.is_system_admin
    or self.is_domain_admin
    or self.hq_api_id == SQLTelerivetBackend.get_api_id()
)
```

Reply:
Yep. That is much clearer to read. Thanks, @millerdev @mkangia for your inputs. I see that there was an explicit test that was added to ensure that domain admins do not edit Twilio Backend which is failing in this case. 
I will ask Product for the inputs on the tickets and proceed accordingly.